Sources/GarageRequestType.swift

```diff
@@ -4,6 +4,7 @@ import APIKit
 public protocol GarageRequestParameterContainer {
     var method: HTTPMethod { get }
     var path: String { get }
+    var rootKeyPath: [String]? { get }
```

Comment:
Rename to `decodeRootKeyPath` and change variable type.

https://github.com/slightair/GarageClientSwift/pull/3/files#diff-29b941133987425a8183ef2f69e3273cR8